tcms/rpc/api/environment.py

```diff
@@ -110,3 +111,27 @@ def filter(query=None):  # pylint: disable=redefined-builtin
         .order_by("name", "description")
         .distinct()
     )
+
+
+@permissions_required("testruns.add_environment")
+@rpc_method(name="Environment.add_environment")
```

Comment:
Should be named `Environment.create` for consistency with the rest of the API.

Reply:
Done in https://github.com/kiwitcms/Kiwi/pull/3082/commits/a1e6436b01353a29ed8dbe31e146b6eb308bf693